Clang build fails: 

```
In file included from src/preferences/dialog/dlgprefcontrols.cpp:27:
src/preferences/dialog/dlgprefcontrols.h:34:11: error: redefinition of 'Duration' as different kind of symbol
namespace Duration {
          ^
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.11.sdk/usr/include/MacTypes.h:286:41: note: previous definition is here
typedef SInt32                          Duration;
                                        ^
```

--- a/src/preferences/dialog/dlgprefcontrols.cpp
+++ b/src/preferences/dialog/dlgprefcontrols.cpp
@@ -72,21 +72,21 @@ DlgPrefControls::DlgPrefControls(QWidget * parent, MixxxMainWindow * mixxx,
 
     double positionDisplayType = m_pConfig->getValue(
             ConfigKey("[Controls]", "PositionDisplay"),
-            static_cast<double>(Duration::DisplayMode::Elapsed));
+            static_cast<double>(TrackTime::DisplayMode::Elapsed));
     if (positionDisplayType ==
-            static_cast<double>(Duration::DisplayMode::Remaining)) {
+            static_cast<double>(TrackTime::DisplayMode::Remaining)) {
         radioButtonRemaining->setChecked(true);
         m_pControlTrackTimeDisplay->set(
-            static_cast<double>(Duration::DisplayMode::Remaining));
+            static_cast<double>(TrackTime::DisplayMode::Remaining));
     } else if (positionDisplayType ==
-                   static_cast<double>(Duration::DisplayMode::ElapsedAndRemaining)) {
+                   static_cast<double>(TrackTime::DisplayMode::ElapsedAndRemaining)) {
         radioButtonElapsedAndRemaining->setChecked(true);
         m_pControlTrackTimeDisplay->set(
-            static_cast<double>(Duration::DisplayMode::ElapsedAndRemaining));
+            static_cast<double>(TrackTime::DisplayMode::ElapsedAndRemaining));
     } else {
         radioButtonElapsed->setChecked(true);
         m_pControlTrackTimeDisplay->set(
-            static_cast<double>(Duration::DisplayMode::Elapsed));
+            static_cast<double>(TrackTime::DisplayMode::Elapsed));
     }
     connect(buttonGroupTrackTime, SIGNAL(buttonClicked(QAbstractButton*)),
             this, SLOT(slotSetTrackTimeDisplay(QAbstractButton *)));
@@ -615,11 +615,11 @@ void DlgPrefControls::slotSetSkin(int) {
 void DlgPrefControls::slotSetTrackTimeDisplay(QAbstractButton* b) {
     double timeDisplay;
     if (b == radioButtonRemaining) {
-        timeDisplay = static_cast<double>(Duration::DisplayMode::Remaining);
+        timeDisplay = static_cast<double>(TrackTime::DisplayMode::Remaining);
     } else if (b == radioButtonElapsedAndRemaining) {
-        timeDisplay = static_cast<double>(Duration::DisplayMode::ElapsedAndRemaining);
+        timeDisplay = static_cast<double>(TrackTime::DisplayMode::ElapsedAndRemaining);
     } else {
-        timeDisplay = static_cast<double>(Duration::DisplayMode::Elapsed);
+        timeDisplay = static_cast<double>(TrackTime::DisplayMode::Elapsed);
     }
     m_pConfig->set(ConfigKey("[Controls]","PositionDisplay"), ConfigValue(timeDisplay));
     m_pControlTrackTimeDisplay->set(timeDisplay);

--- a/src/preferences/dialog/dlgprefcontrols.cpp
+++ b/src/preferences/dialog/dlgprefcontrols.cpp
@@ -67,8 +67,10 @@ DlgPrefControls::DlgPrefControls(QWidget * parent, MixxxMainWindow * mixxx,
             this, SLOT(slotSetTrackTimeDisplay(double)));
 
     // If not present in the config, set the default value
-    if (!m_pConfig->exists(ConfigKey("[Controls]","PositionDisplay")))
-        m_pConfig->set(ConfigKey("[Controls]","PositionDisplay"),ConfigValue(0));
+    if (!m_pConfig->exists(ConfigKey("[Controls]","PositionDisplay"))) {
+        m_pConfig->set(ConfigKey("[Controls]","PositionDisplay"),
+          QString::number(static_cast<int>(TrackTime::DisplayMode::Remaining)));
+    }
 
     double positionDisplayType = m_pConfig->getValue(
             ConfigKey("[Controls]", "PositionDisplay"),

--- a/src/preferences/dialog/dlgprefcontrols.h
+++ b/src/preferences/dialog/dlgprefcontrols.h
@@ -31,7 +31,7 @@ class PlayerManager;
 class MixxxMainWindow;
 class ControlObject;
 
-namespace Duration {
+namespace TrackTime {
     enum class DisplayMode {
         Elapsed,
         Remaining,

--- a/src/widget/wnumberpos.cpp
+++ b/src/widget/wnumberpos.cpp
@@ -53,12 +53,12 @@ void WNumberPos::mousePressEvent(QMouseEvent* pEvent) {
 
     if (leftClick) {
         // Cycle through display modes
-        if (m_displayMode == Duration::DisplayMode::Elapsed) {
-            m_displayMode = Duration::DisplayMode::Remaining;
-        } else if (m_displayMode == Duration::DisplayMode::Remaining) {
-            m_displayMode = Duration::DisplayMode::ElapsedAndRemaining;
-        } else if (m_displayMode == Duration::DisplayMode::ElapsedAndRemaining) {
-            m_displayMode = Duration::DisplayMode::Elapsed;
+        if (m_displayMode == TrackTime::DisplayMode::Elapsed) {
+            m_displayMode = TrackTime::DisplayMode::Remaining;
+        } else if (m_displayMode == TrackTime::DisplayMode::Remaining) {
+            m_displayMode = TrackTime::DisplayMode::ElapsedAndRemaining;
+        } else if (m_displayMode == TrackTime::DisplayMode::ElapsedAndRemaining) {
+            m_displayMode = TrackTime::DisplayMode::Elapsed;
         }
 
         m_pShowTrackTimeRemaining->set(static_cast<double>(m_displayMode));
@@ -94,13 +94,13 @@ void WNumberPos::slotSetPosition(double dPosition) {
         double dDurationMillis = dDurationSeconds * 1000.0;
         double dPosMillis = dPosition * dDurationMillis;
         dPosSecondsElapsed = dPosMillis / 1000.0;
-        if (m_displayMode != Duration::DisplayMode::Elapsed) {
+        if (m_displayMode != TrackTime::DisplayMode::Elapsed) {
             double dPosMillisRemaining = math_max(dDurationMillis - dPosMillis, 0.0);
             dPosSecondsRemaining = dPosMillisRemaining / 1000.0;
         }
     }
 
-    if (m_displayMode == Duration::DisplayMode::Elapsed) {
+    if (m_displayMode == TrackTime::DisplayMode::Elapsed) {
         if (dPosSecondsElapsed >= 0.0) {
             setText(mixxx::Duration::formatSeconds(
                         dPosSecondsElapsed, mixxx::Duration::Precision::CENTISECONDS));
@@ -108,10 +108,10 @@ void WNumberPos::slotSetPosition(double dPosition) {
             setText(QLatin1String("-") % mixxx::Duration::formatSeconds(
                         -dPosSecondsElapsed, mixxx::Duration::Precision::CENTISECONDS));
         }
-    } else if (m_displayMode == Duration::DisplayMode::Remaining) {
+    } else if (m_displayMode == TrackTime::DisplayMode::Remaining) {
         setText(QLatin1String("-") % mixxx::Duration::formatSeconds(
                     dPosSecondsRemaining, mixxx::Duration::Precision::CENTISECONDS));
-    } else if (m_displayMode == Duration::DisplayMode::ElapsedAndRemaining) {
+    } else if (m_displayMode == TrackTime::DisplayMode::ElapsedAndRemaining) {
         if (dPosSecondsElapsed >= 0.0) {
             setText(mixxx::Duration::formatSeconds(
                         dPosSecondsElapsed, mixxx::Duration::Precision::CENTISECONDS)
@@ -130,11 +130,11 @@ void WNumberPos::slotSetPosition(double dPosition) {
 
 void WNumberPos::slotSetDisplayMode(double remain) {
     if (remain == 1.0) {
-        m_displayMode = Duration::DisplayMode::Remaining;
+        m_displayMode = TrackTime::DisplayMode::Remaining;
     } else if (remain == 2.0) {
-        m_displayMode = Duration::DisplayMode::ElapsedAndRemaining;
+        m_displayMode = TrackTime::DisplayMode::ElapsedAndRemaining;
     } else {
-        m_displayMode = Duration::DisplayMode::Elapsed;
+        m_displayMode = TrackTime::DisplayMode::Elapsed;
     }
 
     slotSetPosition(m_dOldPosition);

--- a/src/widget/wnumberpos.h
+++ b/src/widget/wnumberpos.h
@@ -27,7 +27,7 @@ class WNumberPos : public WNumber {
 
   private:
 
-    Duration::DisplayMode m_displayMode;
+    TrackTime::DisplayMode m_displayMode;
 
     // Old value set
     double m_dOldPosition;

